noir_stdlib/src/hash/poseidon.nr

```diff
@@ -1,38 +1,38 @@
 mod bn254; // Instantiations of Poseidon for prime field of the same order as BN254
-use crate::field::modulus_num_bits;
+use crate::field_element::modulus_num_bits;
```

Comment:
Changing the import paths is my main worry in this PR.

Maybe we can make "field" just a reserved word / pseudo-keyword like "self" instead? That way we can still allow it in import paths. Alternatively I suppose it can stay a keyword but we'd have to change the parser to allow paths with `field` in them.